src/Umbraco.Web.BackOffice/Controllers/MemberTypeController.cs

```diff
@@ -150,17 +152,16 @@ public IActionResult DeleteById(int id)
     ///     be looked up via the db, they need to be passed in.
     /// </param>
     /// <returns></returns>
-    public IActionResult GetAvailableCompositeMemberTypes(
-        int contentTypeId,
-        [FromQuery] string[] filterContentTypes,
-        [FromQuery] string[] filterPropertyTypes)
+    [HttpPost]
+    [Authorize(Policy = AuthorizationPolicies.TreeAccessMediaTypes)]
```

Comment:
You don't need to add this policy here, this entire controller is already marked with this policy: 

```
[PluginController(Constants.Web.Mvc.BackOfficeApiArea)]
[Authorize(Policy = AuthorizationPolicies.TreeAccessMemberTypes)]
[ParameterSwapControllerActionSelector(nameof(GetById), "id", typeof(int), typeof(Guid), typeof(Udi))]
public class MemberTypeController : ContentTypeControllerBase<IMemberType>
```

So adding the attribute again for the method is redundant.